src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restriction place where only delegated administrator can access.
+This place is called the Vault. Messages in the Vault will be deleted permanently by a configured retention policy.
```

Comment:
> Messages in the Vault will be deleted permanently by a configured retention policy.

Messages in the Vault will be deleted permanently according to a configurable retention policy.

BTW, is it the case right now? What is the risk we write this sentence and never do that part of the implementation?

Reply:
removed that sentence.